File: help/pt_pt/exercicios_da_ficha_4/Ficha04_017/Ficha04_017.c
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//Main Function
int main() {
    printf("Nota \t: ");
    int nota;
    scanf("%d", &nota);
    if(nota < 0 || nota > 20 ){
        printf("Nota inválida");
    }else{
        printf("Nota válida");
    }
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:33

File: help/pt_pt/exercicios_da_ficha_4/Ficha04_017/Ficha04_017.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha04_017 {

    //Main Function
    static void Main() {
        Console.Write("Nota \t: " );
        int nota = int.Parse(Console.ReadLine());
        if(nota < 0 || nota > 20 ){
            Console.Write("Nota inválida" );
        }else{
            Console.Write("Nota válida" );
        }
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:33
